bedrock/firefox/templates/firefox/set-as-default/landing-page.html

```diff
@@ -0,0 +1,44 @@
+
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_73" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% block page_title_prefix %}{{_('Make Firefox your default browser')}} — {% endblock %}
+{% block page_desc %}{{ _('Choose the browser that protects your privacy. Set Firefox as your default browser.') }}{% endblock %}
+
+{% set utms = '' %} 
```

Comment:
TODO: utms

Reply:
Looks like there are no external links on this landing page, so I think we can drop needing the utms here.